doc/nrf/ug_thread_commissioning_configuring.rst

```diff
@@ -0,0 +1,175 @@
+﻿.. _thread_ot_commissioning_configuring_on-mesh:
+
+Configuring on-mesh Thread commissioning with CLI sample
+########################################################
+
+You can configure on-mesh Thread commissioning using the :ref:`Thread CLI sample <ot_cli_sample>` with two devices to form a Thread network.
+One device will act as a Commissioner and the other will be a Joiner.
+
+.. note::
+    Before you start the configuration process, make sure you are familiar with :ref:`Thread commissioning concepts <thread_ot_commissioning>`, especially :ref:`thread_ot_commissioning_types_on-mesh`.
+
+.. _thread_ot_commissioning_configuring_on-mesh_cli_requirements:
+
+Requirements
+============
+
+* Two of the following development kits:
+
+  * |nRF52840DK|
+  * |nRF52833DK|
+
+.. _thread_ot_commissioning_configuring_on-mesh_cli_flashing:
+
+Flashing the boards
+===================
+
+Flash both development kits with the :ref:`Thread CLI sample <ot_cli_sample>`.
+See the sample's page for details.
+
+After flashing, both devices are pre-commissioned to the same Thread network.
+This means that if ``channel`` and ``panid`` are set to the same value on both devices, they both become members of the same Thread network after entering the following commands:
+
+.. code-block:: console
+
+   > ifconfig up
+   Done
+   > thread start
+   Done
+
+To properly observe the commissioning process, it is recommended to form a new Thread network.
+
+.. _thread_ot_commissioning_configuring_on-mesh_cli_forming:
+
+Forming a new network
+=====================
+
+To form a new Thread network, create a new dataset on one of the devices and set it as active by entering the following commands:
+
+.. code-block:: console
+
+   > ot dataset init new
+   Done
+   > ot dataset commit active
+   Done
+
+To view the newly generated network settings, enter the ``dataset`` command, for example:
+
+.. code-block:: console
+
+   > ot dataset
+   Active Timestamp: 1
+   Channel: 23
+   Channel Mask: 07fff800
+   Ext PAN ID: 36dd32babd209538
+   Mesh Local Prefix: fd51:51f2:fb58:c849/64
+   Master Key: 0278f75cb81f04834f09b5fc095852d6
+   Network Name: OpenThread-8299
+   PAN ID: 0x8299
+   PSKc: 658f3f958bade7db07a36c3fbf2fa2c9
+   Security Policy: 0, onrcb
+   Done
+
+Both devices have now different network settings, which means that it is impossible for either device to communicate with the other.
+This allows for adding a device to the network using on-mesh commissioning.
+
+.. _thread_ot_commissioning_configuring_on-mesh_cli_commissioning:
+
+Performing on-mesh commissioning
+================================
+
+Complete the following steps:
```

Comment:
Clearly indicate in which device the user needs to apply every subsequent command?